meta-resin-common/recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -1,8 +1,4 @@
-PACKAGES =+ "${PN}-bcm43143 ${PN}-rtl8188eu"
-
-FILES_${PN}-bcm43143 = " \
-    /lib/firmware/brcm/brcmfmac43143*.bin \
-    "
+PACKAGES =+ "${PN}-rtl8188eu"
```

Comment:
I'd recommend change-type: minor

Reply:
Changed

Reply:
Also, this needs a few changes to the device repos. currently doing those changes and will ping you back when it's all working on devices side